features/background-position.yml

```
@@ -0,0 +1,22 @@
+name: background-position
+description: The `background-position` CSS property sets the initial position of background images relative to the background origin.
```

I'm not 100% certain about this one, but I kinda think of this as the property that offsets the background.

```suggestion
description: The `background-position` CSS property offsets the initial position of background images relative to the background origin.
```